### DISCUSSION TITLE
Gamecube rumble?

### DISCUSSION THREAD

**Discussion by Mrak408** [NicoHood/Nintendo#58] [Q&A]
Hello! I'm working on using this library to make a "DIY Wavebird" using 2 Arduino Nanos with NRF24L01 modules built in (RF Nano). My code does indeed work, but I'm wondering if it's possible to incorporate rumble support in a way that it acts just like a wired controller. As in, if you get hit in Smash Melee, the controller would vibrate. I am not a programmer so sorry if I missed something obvious. Thanks!

**Comment by dav0163** [comment]
> Hello! I'm working on using this library to make a "DIY Wavebird" using 2 Arduino Nanos with NRF24L01 modules built in (RF Nano). My code does indeed work, but I'm wondering if it's possible to incorporate rumble support in a way that it acts just like a wired controller. As in, if you get hit in Smash Melee, the controller would vibrate. I am not a programmer so sorry if I missed something obvious. Thanks!

Hey man i’m having problems getting the arduino nanos to pickup on the gamecube controller. Have you had this issue before?
